lib/location/method_gnss.c

```diff
@@ -110,6 +110,7 @@ static void method_gnss_pgps_ext_work_fn(struct k_work *item);
 #endif
 
 static int fixes_remaining;
+static int blocked_epoch_count;
```

Comment:
I'd like to see a comment for this parameter as it's not evident.
I'm also thinking about the name as it doesn't ring a bell on what it would be. I'm thinking now `gnss_blocked_count` etc. but please discuss with @tokangas about the proper name. Maybe the current one is something you've already agreed on.

Reply:
The word "epoch" is not used elsewhere in our libraries or documentation, so it's probably not evident for everybody what it means (although it's a valid GNSS term). And again I'm slightly reserved about the word "blocked". 😄

Reply:
Renamed to insuf_timewin_count and added comment.